### PR TITLE
[com_banners] Add search tooltip to all views

### PR DESCRIPTION
**Pull request by andrepereiradasilva**
#### Summary of Changes

Simple PR to improve the search tooltip and element label (accessibility) in com_banners.
#### Testing Instructions

Code review, or:
1. Apply patch
2. Go to Components -> Banners -> Banners , check the tooltip in the search field.
3. Go to Components -> Banners -> Clients, check the tooltip in the search field.
4. Go to Components -> Banners -> Tracks, check the tooltip in the search field.

### CODE REVIEW

File: administrator/language/en-GB/en-GB.com_banners.ini [hunk 1]
```diff
@@ -6,6 +6,8 @@
 COM_BANNERS="Banners"
 COM_BANNERS_BANNER_DETAILS="Details"
 COM_BANNERS_BANNER_SAVE_SUCCESS="Banner successfully saved."
+COM_BANNERS_BANNERS_FILTER_SEARCH_DESC="Search in banner name and alias. Prefix with ID: to search for an banner ID."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_banners.ini]: change to 
a banner

The rule is that it is always A unless the next word begins with a vowel. (there are exceptions of course;) )

File: administrator/language/en-GB/en-GB.com_banners.ini [hunk 2]
```diff
@@ -35,6 +37,8 @@ COM_BANNERS_BEGIN_LABEL="Begin Date"
 COM_BANNERS_CANCEL="Cancel"
 COM_BANNERS_CLICK="Click"
 COM_BANNERS_CLIENT_SAVE_SUCCESS="Client successfully saved."
+COM_BANNERS_CLIENTS_FILTER_SEARCH_DESC="Search in client name. Prefix with ID: to search for an client ID."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_banners.ini]: change to 
a client

**Comment by brianteeman** [comment]
Sorry a few en-GB comments inline

**Comment by andrepereiradasilva** [comment]
done

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on c5edf2371fc859d64ce5567bdbdcf2e77269513e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10175">issues.joomla.org/joomla-cms/10175</a>.</sub>

**Comment by MATsxm** [comment]
I have tested this item :white_check_mark: successfully on c5edf2371fc859d64ce5567bdbdcf2e77269513e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10175">issues.joomla.org/joomla-cms/10175</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10175">issues.joomla.org/joomla-cms/10175</a>.</sub>

**Pull request merged**